e2echart/test-risk-analysis.html

```diff
@@ -0,0 +1,106 @@
+<html lang="en">
+<head>
+    <title>TEST_RISK_ANALYSIS_TITLE_GOES_HERE</title>
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+</head>
+<body onLoad="buildTestCaseTable('#test_case_results')">
+<h1>
+</h1>
+<p>
+    "Risk analysis is performed by Sippy[<a href="TEST_RISK_ANALYSIS_SIPPY_URL_GOES_HERE">link</a>] to attempt to
+    determine if the failures in this job are abnormal when compared to results for similar jobs over the past week,
+    and amidst on-going incidents in the CI infrastructure. Risk analysis API will not catch everything and is a
+    relatively simple implementation today, please reach out to the Technical Release Team if you spot abnormalities
+    or have suggestions."
+</p>
```

Comment:
Spyglass supports "info" tooltips, if you'd prefer to put most of this info except maybe the link in there.  Example:

https://github.com/openshift/release/blob/9548766e2f234917b40732d279fa1fe6cf7f6dad/ci-operator/step-registry/gather/extra/gather-extra-commands.sh#L577-L579

Reply:
Does the tooltip support links? If one puts the cursor on and get the tooltip, can he click on the link to get to sippy? I just experimented and it doesn't seem to support that.

Reply:
Right, I suggested not putting the link.  You can leave it here if you want, but it'd be more consistent if we described what the page is doing in the tooltip.

Reply:
Devan suggested the link. @dgoodwin do you prefer this text in tooltip without the link to Sippy, or a paragraph on the top of the page?

Reply:
My thought was here was just to link to top level Sippy page, so a user could see what it was. Not directly to risk analysis. I can live without either though if anyone feels strongly.

And if we keep it, I'd just do the word Sippy as the link, not a [] after it, that was just markdown slipping into my suggestions for the card.

Reply:
Sorry if I wasn't clear, I just meant putting the link outside of the tooltip somewhere on the page would be fine.

Reply:
> Is there a corresponding PR to openshift/release to add this as an [HTML page spyglass should show](https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml#L108-L114)? There's no reason it can't be there now so we could see the results in the presubmits

@deepsm007 Deep has a PR here: https://github.com/openshift/release/pull/33731

You meant that PR can be merged now so that we can see the spyglass in our job runs. Good point.